app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+    @ingredients = @recipe.ingredients
```

Comment:
It would be a little more MVC friendly to just call `@recipe.ingredients` in the view rather than defining an extra instance variable in the controller. We want to keep our controllers as thin as possible (without performing data or action based logic in the view).